cmsmon-tools.spec

```diff
@@ -1,44 +1,44 @@
-### RPM cms cmsmon-tools 0.5.32
+### RPM cms cmsmon-tools 0.5.35
 ## NOCOMPILER
 
 %define arch linux-amd64
-%define promv 2.25.1
-%define amver 0.21.0
+%define promv 2.28.0
+%define amver 0.22.2
 %define sternv 1.11.0
-%define apsver 0.1.85
+%define apsver 0.1.106
+%define trivyver 0.18.3
+%define gover 0.16.5
+%define heyver 0.0.2
+%define k8s_info_ver 0.0.1
 %define monit_commands monit ggus_parser alert annotationManager nats-sub nats-pub dbs_vm
-%define common_commands promtool amtool prometheus hey stern
+%define common_commands promtool amtool prometheus hey stern trivy k8s_info
 %define flags -ldflags="-s -w -extldflags -static" -p %{compiling_processes}
 Source0: https://github.com/dmwm/CMSMonitoring/releases/download/%{realversion}/cmsmon-tools.tar.gz
 Source1: https://github.com/prometheus/prometheus/releases/download/v%promv/prometheus-%promv.linux-amd64.tar.gz
 Source2: https://github.com/prometheus/alertmanager/releases/download/v%amver/alertmanager-%amver.linux-amd64.tar.gz
-Source3: https://github.com/vkuznet/hey/archive/x509-csv-fixes.tar.gz
+Source3: https://github.com/vkuznet/hey/releases/download/%heyver/hey-tools.tar.gz
 Source4: https://github.com/wercker/stern/releases/download/%sternv/stern_linux_amd64
 Source5: https://github.com/vkuznet/auth-proxy-server/releases/download/%apsver/auth-proxy-tools.tar.gz
-Source6: https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/tools/k8s_info.go
+Source6: https://github.com/vkuznet/k8s_info/releases/download/%k8s_info_ver/k8s_info-tools.tar.gz
+Source7: https://github.com/aquasecurity/trivy/releases/download/v%trivyver/trivy_%{trivyver}_Linux-64bit.tar.gz
 
 BuildRequires: go
 
 # RPM macros documentation
 # http://www.rpm.org/max-rpm/s1-rpm-inside-macros.html
+# http://ftp.rpm.org/max-rpm/s1-rpm-inside-macros.html
 %prep
 %setup -D -T -b 0 -n cmsmon-tools
 %setup -D -T -b 1 -n prometheus-%promv.%arch
-%setup -D -T -b 3 -n hey-x509-csv-fixes
+%setup -D -T -b 3 -n hey-tools
 %setup -D -T -b 2 -n alertmanager-%amver.%arch
 %setup -D -T -b 5 -n auth-proxy-tools
+%setup -D -T -b 6 -n k8s_info-tools
+%setup -D -T -b 7 -n trivy-%trivyver -c trivy-%trivyver
```

Comment:
may be it should be `-a 7` , let me try this.

Reply:
@smuzaffar it seems very strange to me since according to the log the tar ball was fetched and untarred, see
```
+ cd /data/cmsbld/jenkins/workspace/build-any-ib/w/BUILD/cc8_amd64_gcc9/cms/cmsmon-tools/0.5.35-3468a0175746113b8a4ac4a4a14fe6a9
+ gzip -dc /data/cmsbld/jenkins/workspace/build-any-ib/w/SOURCES/cms/cmsmon-tools/0.5.35-3468a0175746113b8a4ac4a4a14fe6a9/trivy_0.18.3_Linux-64bit.tar.gz
+ tar -xvvof -
-rw-r--r-- runner/docker 11357 2021-05-24 07:45 LICENSE
-rw-r--r-- runner/docker  6421 2021-05-24 07:45 README.md
-rw-r--r-- runner/docker  2976 2021-05-24 07:45 contrib/asff.tpl
drwxr-xr-x                  Creating directory: contrib
-rw-r--r-- runner/docker  2514 2021-05-24 07:45 contrib/gitlab.tpl
-rw-r--r-- runner/docker  4156 2021-05-24 07:45 contrib/html.tpl
-rw-r--r-- runner/docker   739 2021-05-24 07:45 contrib/junit.tpl
-rw-r--r-- runner/docker  3578 2021-05-24 07:45 contrib/sarif.tpl
-rwxr-xr-x runner/docker 34869248 2021-05-24 07:50 trivy
```
Therefore, I would expect `trivy` to be present on file system. Can it be local FS issue? Can you trigger build again to see if issue persists. Also, if you have access to build area can you just list what is available in build area.